80_machinelearning/ML1000_Machine_Learning.md

```
@@ -0,0 +1,65 @@
+
+Denne video kan være lidt overvældende, hvis du næsten ikke ved noget om 
+maskinlæring/AI.   
+Det er bedst at se videoen igen, når du har arbejdet dig igennem de følgende 
+tutorials.   
+Så vil du være i stand til at kategorisere mange af de termer, der nævnes i 
+videoen.   
+Det vil give dig en bedre forståelse af, hvilke områder af maskinlæring/AI disse tutorials dækker, og hvad der ellers er at opdage.
+
+
+
+### Gennemfør disse tutorials:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)